plugins/woocommerce/client/admin/client/settings-payments/index.tsx

```diff
@@ -199,6 +201,9 @@ export const SettingsPaymentsMethods = () => {
 					( paymentMethod ) => ! paymentMethodsState[ paymentMethod ]
 				)
 				.join( ', ' ),
+			store_country:
```

Comment:
Nitpick: Do you think it makes sense to add a default country if it is not set, e.g., the US?